react/week-40/my-react-app/playwright.config.js

```diff
@@ -39,15 +39,15 @@ export default defineConfig({
       use: { ...devices['Desktop Chrome'] },
     },
 
-    {
-      name: 'firefox',
-      use: { ...devices['Desktop Firefox'] },
-    },
+    // {
+    //   name: 'firefox',
+    //   use: { ...devices['Desktop Firefox'] },
+    // },
 
-    {
-      name: 'webkit',
-      use: { ...devices['Desktop Safari'] },
-    },
+    // {
```

Comment:
Varför tog du bort webkit? Vi behöver det!!!

Reply:
Okej, tack just nu kom vi överens om med It projektledarna att det inte finns behov och tid för det.